tests/python/unittest/test_ndarray.py

```diff
@@ -862,7 +862,8 @@ def test_iter():
     for i in range(x.size):
         assert same(y[i].asnumpy(), x[i].asnumpy())
 
-@unittest.skip("test fails intermittently. temporarily disabled till it gets fixed. tracked at https://github.com/apache/incubator-mxnet/issues/8049")
+# @vandanavk: tracked at https://github.com/apache/incubator-mxnet/issues/8049
```

Comment:
Feel free to remove those comments and simply link the issue in PR description.

Reply:
Resubmitted after removing this comment